charts/flyte-core/values.yaml

```diff
@@ -487,8 +487,7 @@ configmap:
     # -- resource manager configuration
     propeller:
       resourcemanager:
-        type:
-
+        type: redis
```

Comment:
I think the default one shouldn't rely on redis... it should just be `noop`
https://github.com/flyteorg/flytepropeller/blob/eaf084934de5d630cd4c11aae15ecae780cc787e/pkg/controller/nodes/task/resourcemanager/config/config.go#L14